src/components/Icons/Logout/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const LogOutSVG = ({
```

Comment:
Weet niet waar je het logo gehaald hebt, maar ik ben wel fan van [heroicons](https://heroicons.com/). Ze hebben ook een React package. Is van de makers van Tailwind, je kan er gewoon een className aan meegeven voor kleur etc. Nog wel handig, maar zeker geen must :)

Reply:
Ik heb gehaald  via [Feather icons](https://feathericons.com/). Mijn persoonlijke favoriet 😄